java/core/pom.xml

```diff
@@ -8,7 +8,7 @@
     <parent>
         <groupId>com.lancedb</groupId>
         <artifactId>lance-parent</artifactId>
-        <version>0.1-SNAPSHOT</version>
```

Comment:
We do autobump the version during release process. Will it bump this one as well?

Reply:
no, need some modifications on the release process to bump up the maven versions